src/clld_markdown_plugin/__init__.py

```diff
@@ -141,13 +150,6 @@ def repl(ml):
         return ml
 
     md = Markdown(
-        extensions=[
-            TocExtension(permalink=permalink),
-            "markdown.extensions.fenced_code",
-            "markdown.extensions.md_in_html",
-            "markdown.extensions.tables",
-            "markdown.extensions.attr_list",
-            "markdown.extensions.footnotes",
-        ]
+        extensions=[TocExtension(permalink=permalink)] + settings["extensions"]
```

Comment:
I think extensions that need parameters should always be specified as custom extension in the settings (initialized python objects should work). Otherwise we'd make the call signature of `markdown` too intransparent.

Reply:
So the `permalink` keyword of `markdown` could be removed as well.

Reply:
Oh, that's only there as a leftover from [this comment](https://github.com/clld/clld-markdown-plugin/issues/1#issuecomment-1074971828). Will remove it.